assets/styles/code-title.scss

```diff
@@ -0,0 +1,14 @@
+.code-title {
+  color: rgba(238, 130, 238, 0.87);
+  background-color: rgba(68, 74, 90, 0.53);
```

Comment:
any chance these can use the css vars defined in https://github.com/jackyzha0/quartz/blob/hugo/assets/styles/custom.scss#L2-L13 ?

Reply:
Ideally yes, but it will look a bit off when switching themes as the syntax theme doesn't invert (yet). I might have a PR coming in the future to address that however (#149).

Reply:
I will be looking into the separate syntax coloring another day, but I think given the scope of this PR, this is resolved. Using any variables from the `custom.scss` would look strange when toggling dark mode, as the syntax coloring under it will not toggle between dark and light (yet) 👍🏻